src/tests/profiling/test_foundry_prove.py

```diff
@@ -28,15 +29,15 @@
 
 def test_foundy_prove(profile: Profiler, use_booster: bool, bug_report: BugReport | None, tmp_path: Path) -> None:
     foundry_root = tmp_path / 'foundry'
-
     _forge_build(foundry_root)
+    foundry = load_foundry(foundry_root=foundry_root)
```

Comment:
Consider returning `Foundry` from `_forge_build`. It's fine to instantiate it with the constructor directly there.

Reply:
Done!